sdk/src/manifest.rs

```diff
@@ -2002,19 +2012,19 @@ pub(crate) mod tests {
         let signer = temp_remote_signer();
 
         // Embed a manifest using the signer.
-        let (out_vec, _out_manifest) = manifest
+        let (out_vec, out_manifest) = manifest
             .embed_from_memory_remote_signed("image/webp", image, signer.as_ref())
             .await
             .expect("embed_stream");
 
         // try to load the image
-        let manifest_store = Reader::from_stream("image/webp", Cursor::new(out_vec)).unwrap();
+        let manifest_store = Reader::from_stream_async("image/webp", Cursor::new(out_vec))
+            .await
+            .unwrap();
 
-        /* to be enabled later
-                // try to load the manifest
-                let mut validation_log = DetailedStatusTracker::new();
-                Store::from_jumbf(&out_manifest, &mut validation_log).expect("manifest_load_error");
-        */
+        // try to load the manifest
+        let mut validation_log = DetailedStatusTracker::new();
+        Store::from_jumbf(&out_manifest, &mut validation_log).expect("manifest_load_error");
```

Comment:
I don't think we need to call both Reader::from_stream.. and Store::from_jumbf, since the first one calls the second one.

Reply:
I will remove those.